### PR TITLE
Add uwsift package

### PR DESCRIPTION
**Pull request by djhoese**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

CC @rayg-ssec (please comment that you are OK being a maintainer)

### CODE REVIEW

**Comment by rayg-ssec** [comment]
I consent to being listed as a maintainer.

**Comment by djhoese** [comment]
@conda-forge/staged-recipes This recipe is ready for review. Let me know if you have any questions. Thank you!

File: recipes/uwsift/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{% set name = "uwsift" %}
+{% set version = "1.1.0a1" %}
```
Comment (beckermr) [inline recipes/uwsift/meta.yaml]: conda-forge generally does not support a dev release. Is there another version?

Comment (djhoese) [reply recipes/uwsift/meta.yaml]: There are older versions of the library and project but they are not compatible with conda-forge's current offerings. I am hoping to get a release out in the next couple weeks to a month and was hoping to have beta testers walk through the installation instructions using conda-forge.

If this is really not traditionally done then I will find a better way until a full release is out.

NOTE: I would like to learn about options for "dev" labels for a package on conda-forge if that is a thing for less widely used packages like this.

Comment (beckermr) [reply recipes/uwsift/meta.yaml]: Alright. Let’s ping @conda-forge/staged-recipes and @conda-forge/core for any opinions. Idk the status of dev releases in cases like this.

Comment (djhoese) [reply recipes/uwsift/meta.yaml]: I've now released a 1.1.0 release of the uwsift package. Once the builds pass this should be good to review again.

File: recipes/uwsift/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - pyshp
+    - pyqt
+    - pyqtgraph
+    - pyyaml
+    - rasterio
+    - satpy
+    - scikit-image
+    - shapely
+    - sqlalchemy
+    - vispy
```
Comment (mbargull) [inline recipes/uwsift/meta.yaml]: Acc. to https://github.com/ssec/sift/blob/1.1.0/setup.py#L192-L197:
```suggestion
    - vispy >=0.6
```

File: recipes/uwsift/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - imageio-ffmpeg
+    - matplotlib
+    - netcdf4
+    - numba
+    - numpy
+    - pillow
+    - pygrib  # [not win]
+    - pyproj
+    - pyshp
+    - pyqt
```
Comment (mbargull) [inline recipes/uwsift/meta.yaml]: Acc. to https://github.com/ssec/sift/blob/1.1.0/setup.py#L192-L197:
```suggestion
    - pyqt5 >=5.9
```

File: recipes/uwsift/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    - sqlalchemy
+    - vispy
+
+test:
+  imports:
+    - uwsift
+
+about:
+  home: http://sift.ssec.wisc.edu/
+  license: GPL-3.0
```
Comment (mbargull) [inline recipes/uwsift/meta.yaml]:
```suggestion
  license: GPL-3.0-or-later
```

**Comment by djhoese** [comment]
Thanks @mbargull! Very thorough. I've made the changes you suggested.

**Comment by mbargull** [comment]
@djhoese, thanks for your contribution and patience!

**Pull request merged**